scripts/contentScript.js

```diff
@@ -139,15 +139,82 @@ async function getHighlightedPR(repoOwner, repoName, userId) {
 };
 
 // for showing all tracked/ untrack pr in a organization
-async function getTrackedPR(orgName) {
+async function getRepoInfo(orgName) {
 	const body = { "org": `${orgName}` }
 	const url = 'https://gcscruncsql-k7jns52mtq-el.a.run.app/setup/repos';
 	const trackedRepos = await apiCall(url, body);
-	if(trackedRepos){
-		cssForGithubTrackRepo(trackedRepos);
+	if (trackedRepos) {
+		return trackedRepos;
 	};
 }
 
+async function getTrackedRepo(orgName) {
+	const data = await getRepoInfo(orgName);
+	if (data) cssForGithubTrackRepo(data);
+}
+
+
+// adding favButton
+async function addButton(orgName, orgRepo) {
+
+	const addedRepoList = await getRepoInfo(orgName);
+
+	if (!addedRepoList.includes(orgRepo)) {
+		const PrSection = document.getElementById('repo-content-pjax-container');
+		// for vibinex logo
+		const img = document.createElement("img");
+		img.setAttribute('id', 'vibinexWarningLogo')
+		img.src = "https://vibinex.com/favicon.ico";
+		img.style.width = '35px';
+		img.style.height = '35px';
+		img.style.borderRadius = '35px';
+		img.style.position = 'fixed';
+		img.style.left = '30px';
+		img.style.bottom = '50px';
+		img.style.cursor = 'pointer'
```

Comment:
Very low LOGAF, but can't this be done like this:

```suggestion
		img.style = {
				width: '35px';
				height: '35px';
				borderRadius: '35px';
				position: 'fixed';
				left: '30px';
				bottom: '50px';
				cursor : 'pointer'
		}
```

Reply:
This syntax is not working !

Reply:
Remove the commented code, in that case.